pkg/docker/docker.go

```diff
@@ -669,7 +680,7 @@ func (d *stiDocker) CommitContainer(opts CommitContainerOptions) (string, error)
 			User:   opts.User,
 		}
 		dockerOpts.Run = &config
-		glog.V(2).Infof("Committing container with config: %+v", config)
+		glog.V(2).Infof("Committing container with config: %#v", dockerOpts)
```

Comment:
Since `config` is a pointer, this will just log the memory address of the pointer, not very useful.
This change actually hides the content of `config` from the log output...